index.js

```diff
@@ -1,21 +1,37 @@
 'use strict';
 
+var Configstore = require('configstore');
 var githubRepos = require('github-repositories');
 var githubTokenUser = require('github-token-user');
 var ghGot = require('gh-got');
 var eachAsync = require('each-async');
+var conf = new Configstore('github-add-collab');
```

Comment:
Maybe related to https://github.com/kevva/github-add-collab/pull/6/files#r30256718 ? It doesn't for me.